src/components/Project/components/RewardCard.tsx

```diff
@@ -46,8 +47,16 @@ export const RewardCard: React.FC<RewardCardProps> = ({ className, nft }) => {
           />
         </DialogTrigger>
         <div className="flex flex-col gap-4 p-4 pb-5">
-          <div className="text-base font-medium text-gray-900">
-            {nft.metadata.name}
+          <div className="flex items-center justify-between gap-5">
+            <div className="text-base font-medium text-gray-900">
+              {nft.metadata.name}
+            </div>
+            <DialogTrigger>
+              <div className="flex items-center gap-0.5 text-end text-gray-400 hover:text-blue-400">
```

Comment:
once conflicts resolved, can slap a `group-hover` so it turns blue when any of the clickable card is hovered

```suggestion
              <div className="flex items-center gap-0.5 text-end text-gray-400 group-hover:text-blue-400">
```